decredPoS/decredPoS_script.md

```diff
@@ -37,13 +35,15 @@ After the update threshold is met voting will begin on the *first block* of the
 
 Each rule change interval spans 8,064 blocks (or about 4 weeks) during which votes for any active rule change processes are collected.  A revote occurs if 90% of votes collected during the entire interval abstain.  Else, if 75% of non-abstaining votes signal yes, then the proposed rule changes will be activated at the end of the *next* rule change interval.
 
-After a ticketholder's votes has been included in a block and a 256 block immaturity phase has passed, the price of the voted ticket is fully refunded and a portion of the Decred block reward is credited to their account as compensation for their active participation.
+After the ticketholder's votes has been included in a block and a 256 block immaturity phase has passed, the price of the voted ticket is fully refunded and a portion of the Decred block reward is credited to their account as compensation for their active participation.
```

Comment:
votes have* / vote* has

a / the is a matter of emphasis

Since the immature stage has already been introduced, it could say "another" immature phase rather than mentioning the amount of blocks again?